signus/src/mainmenu.cpp

```diff
@@ -106,8 +106,11 @@ static int ProcessMenu(const char *mask1, const char *mask2) {
 
 	DrawMN(pt1, pt2, bg, sel);
 
-	while (TRUE) {
+	int run = TRUE;
```

Comment:
Move this up to the other variable declarations.

Reply:
Done!